reference/5.1/CimCmdlets/Invoke-CimMethod.md

```diff
@@ -17,71 +17,81 @@ Invokes a method of a CIM class.
 ## SYNTAX
```

Comment:
Won't let me place a comment above for something. The Synopsis Header should be surrounded by new lines

Reply:
No, that is the one exception. If you put a blank line after the Synopsis then the text does not appear in Get-Help.